inorbit_edge/models.py

```diff
@@ -85,28 +85,40 @@ class RobotSessionModel(BaseModel):
 
     * INORBIT_API_KEY (required): The InOrbit API key
     * INORBIT_USE_SSL: If SSL should be used (default is true)
+    * INORBIT_API_URL: The URL of the API (default is
+        inorbit_edge.robot.INORBIT_CLOUD_SDK_ROBOT_CONFIG_URL)
+    * INORBIT_REST_API_URL: The URL of the InOrbit REST API (default is
+        inorbit_edge.robot.INORBIT_REST_API_URL)
 
     Attributes:
         robot_id (str): The unique ID of the robot
         robot_name (str): The name of the robot
-        robot_api_key (str | None, optional): The robot key for InOrbit cloud services
+        robot_key (str | None, optional): The robot key for InOrbit cloud services

```

Comment:
Why rename this? It will break anyone who calls it with explicate args.

Reply:
Oh I changed it because it was wrong, sorry. I will move it to a different MR but `RobotSession` takes `robot_key` as a keyword argument. `robot_api_key`. I believe it is is a mistake.

Reply:
Here: https://github.com/inorbit-ai/edge-sdk-python/blob/fa3d78fef7d5be354b6c13dd79e85ea60d1be559/inorbit_edge/robot.py#L125
`robot_api_key` is a variable that gets populated when the `connect()` method is call. But it is not an argument for `RobotSession`

Reply:
@russell-inorbit I reverted the change. I will create a new PR with that fix.